addons/traefik/1.7.x/tests/00-install.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl apply -f ../traefik-9.yaml
```

Comment:
`v0.3.0` feature would change to 

```
apiVersion: kudo.dev/v1beta1
kind: TestStep
apply:
  - ../traefik-9.yaml 
 ```

example: https://github.com/kudobuilder/kuttl/tree/master/pkg/test/test_data/teststep-apply